docs/platforms/python/integrations/bottle/index.mdx

```diff
@@ -90,6 +91,22 @@ def myendpoint():
 - If you set `transaction_style="endpoint"`, the transaction name is `"myendpoint"`, since that is the route handler function's name.
 - If you set `transaction_style="url"`, the transaction name is `"/myurl/<foo>"`, since that is the URL path.
 
+
+### `failed_request_status_codes`
+
+A `set` of integers that will determine when an [`HTTPResponse`](https://bottlepy.org/docs/dev/api.html#bottle.HTTPResponse), which is raised or returned by the request handler, should be reported to Sentry. The `HTTPResponse` is reported to Sentry if its status code is contained in the `failed_request_status_codes` set.
+
+Examples of valid values for `failed_request_status_codes`:
+
+- `{500}` will only report `HTTPResponse` with status 500.
```

Comment:
```suggestion
- `{500}` will only report an `HTTPResponse` with a 500 status.
```

Reply:
I would prefer to keep this one (and the next comment) as is. This is in alignment with the language used in the [docs for AIOHTTP's](https://github.com/getsentry/sentry-docs/pull/11433) `failed_request_status_codes` option